src/Native/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ add_compile_options(-fPIC)
 add_compile_options(-I${CMAKE_CURRENT_SOURCE_DIR}/Common)
 add_compile_options(-I${CMAKE_CURRENT_BINARY_DIR}/Common)
 add_compile_options(-Wno-c99-extensions)
+add_compile_options(-Wno-switch-enum)
```

Comment:
Hmm, if we are going to keep the struct member, then this is not required.
But if the maintainers want the whole `case` ifdef'd out, then it won't compile without these.
@stephentoub, can you please take a look again?